### PR TITLE
Accounts Controller

### PR DESCRIPTION
**Pull request by Bazik33**

### CODE REVIEW

File: backend/tests/NotePaste.IntegrationTests/Helpers.cs [hunk 1]
```diff
@@ -13,6 +13,15 @@
 
  namespace NotePaste.IntegrationTests
 {
+    public class UserModel
+    {
+        public string id { get; set; }
```
Comment (KanarekLife) [inline backend/tests/NotePaste.IntegrationTests/Helpers.cs:18]: Properties should be Pascal

File: backend/tests/NotePaste.UnitTests/Helpers.cs [hunk 2]
```diff
@@ -0,0 +1,12 @@
+using Google.Protobuf.WellKnownTypes;
+using NotePaste.WebApi.Models.SettingsModels;
+using System;
+using Xunit;
+
+namespace NotePaste.UnitTests
+{
+    public class Helpers
+    {
+
```
Comment (KanarekLife) [inline backend/tests/NotePaste.UnitTests/Helpers.cs:10]: ?

File: backend/tests/NotePaste.IntegrationTests/Helpers.cs (hunk 1 above)
Comment (Bazik33) [reply backend/tests/NotePaste.IntegrationTests/Helpers.cs:18]: Fixed.

File: backend/tests/NotePaste.UnitTests/Helpers.cs (hunk 2 above)
Comment (Bazik33) [reply backend/tests/NotePaste.UnitTests/Helpers.cs:10]: Deleted.

**Comment by Bazik33** [comment]
Closed due to repo recreation.

**Pull request closed without merging**